standards.md

```diff
@@ -265,7 +265,8 @@ PRs must adhere to the project's API stability policy.
   * Validation + validation tests
 * Unit tests
   * Coverage should remain the same or increase
-  * Each test should only test one function at a time
+  * Each test class should only test one function at a time, and each test case should only test
+  one set of inputs at a time
```

Comment:
```suggestion
  * As much as possible, we recommend each test have a focused set of inputs and outputs - e.g.  testing one function at a time, limiting the set of inputs to make it clear what's being tested, etc.
```

I somewhat disagree with this line as written - it comes across as a requirement, when inevitably there are going to be things that need to bend the rule here. I think it's a good rule of thumb and recommendation though. Took a pass a re-writing this to be more "best practice"-y. lmk what you think!

Reply:
Thanks for the feedback! I agree it might have been a bit too prescriptive. I reworded based on your suggestion.